src/i18n/locales/en/electron.json

```diff
@@ -11,4 +11,4 @@
       "restarting": "Restarting..."
     }
   }
-}
```

Comment:
can you undo this change?

Reply:
Somehow I couldn't find this deletion in any of the branches